extensions/amp-story-interactive/0.1/test/test-amp-story-interactive-img-poll.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * Copyright 2021 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {AmpStoryInteractiveImgPoll} from '../amp-story-interactive-img-poll';
+import {Services} from '#service';
+import {addConfigToInteractive} from './test-amp-story-interactive';
+import {measureMutateElementStub} from '#testing/test-helper';
+
+describes.realWin(
```

Comment:
Can you add a test that checks the images passed as options are being used in the corresponding circles?